src/features/retrospective/ui/RetrospectiveView.tsx

```
@@ -23,83 +26,136 @@ const RetrospectiveView: React.FC = () => {
+          <RetrospectiveSummary
+            essential={amounts.essential}
+            conscious={amounts.conscious}
+            emotional={amounts.emotional}
+          />
+        </div>
+        {sortedRetrospectiveCards.map(
+          ({ retrospective, consumption, onClickRetrospectiveDetail }) => (
+            <>
+              <div className='h-3 bg-[#F5F3F0] shrink-0' />
+              <RetrospectiveCard
+                key={retrospective?.consumptionKind}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)